lib/errors_savior/dictionary/local_dictionary.yml

```diff
@@ -0,0 +1,47 @@
+# Model errors (1xxx)
+# -------------------
+
+field_validation_error:
```

Comment:
is the name used for something, if not, wouldn't it be nicer to be the error class directly as the key, like:

```yml
ActiveModel::ValidationError:
  error_code: 1001
  http_status_sym: unprocessable_entity
  http_status_code: 422
  message: One or more fields are invalid
```

Does this also supports I18n ? Is it planned on another version?